soc/arm/silabs_exx32/Kconfig

```diff
@@ -134,6 +134,18 @@ choice SOC_GECKO_EMU_DCDC_MODE
 		bool "Bypass"
 endchoice
 
+config SOC_GECKO_DEV_INIT
+	bool
+	help
+	  Device initialization as per the sequence in simplicity studio.
+	  This is currently supported by efr32bg22 series.
```

Comment:
this option is not clear, it seems to translate to clock configuration. It should be clarified, and implications detailed.

Reply:
I've changed the description of this config option.